website/content/docs/v0.10/Single Board Computers/rock64.md

```diff
@@ -28,8 +28,8 @@ xz -d metal-rock64-arm64.img.xz
 
 ## Writing the Image
 
-The path to your SD card can be found using `fdisk` on Linux or `diskutil` on Mac OS.
-In this example we will assume `/dev/mmcblk0`.
+The path to your SD card can be found using `fdisk` on Linux or `diskutil` on macOS.
+In this example, we will assume `/dev/mmcblk0`.
```

Comment:
I believe comma is not required here but not a native English speaker ;)

Reply:
Grammarly asked me to add it, and who am I to argue with it? 😆